sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -0,0 +1,3270 @@
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the device twin object",
+            "schema": {
+              "$ref": "#/definitions/Twin"
+            }
+          }
+        },
+        "summary": "Replaces tags and desired properties of a module twin."
+      },
+      "patch": {
+        "description": "Updates a module twin. See https://docs.microsoft.com/azure/iot-hub/iot-hub-devguide-device-twins for more information. For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
+        "operationId": "Twin_UpdateModuleTwin",
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "description": "Device ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "mid",
+            "in": "path",
+            "description": "Module ID.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "deviceTwinInfo",
+            "in": "body",
+            "description": "Device twin information",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/Twin"
+            }
+          },
+          {
+            "name": "If-Match",
+            "in": "header",
+            "description": "A string representing a weak ETag for the device twin, as per RFC7232. It determines if the update operation should be carried out.",
+            "required": false,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/api-version"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Returns the device twin object",
+            "schema": {
+              "$ref": "#/definitions/Twin"
+            }
+          }
+        },
+        "summary": "Updates tags and desired properties of a module twin."
+      }
+    },
+    "/digitalTwins/{digitalTwinId}/interfaces": {
+      "get": {
+        "description": " For IoT Hub VNET related features(https://docs.microsoft.com/en-us/azure/iot-hub/virtual-network-support) please use API version '2020-03-13'.These features are currently in general availability in the East US, West US 2, and Southcentral US regions only. We are actively working to expand the availability of these features to all regions by end of month May. For rest of the APIs please continue using API version '2019-10-01'",
```

Comment:
Are we not suggesting they get rid of the "vnet" stuff on these? Or at least, not make it the first part of the description?

Reply:
Hmm, this should have disappeared when I merged in the correct description. Let me double check how that merge worked before you keep reviewing